### PR TITLE
MDOT- Alert for No Products Selected

### PR DESCRIPTION
**Pull request by maharielrosario**
## Description
Display alert if no products were selected

## Testing done
Unit test added

## Screenshots
screenshot-localhost_3001-2020 06 24-21_01_10

## Acceptance criteria
- [x] Design team approval

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -74,94 +75,135 @@ const ConfirmationPage = ({
...
+              or email <a href="mailto:dalc.css@va.gov">dalc.css@va.gov</a>.
             </p>
-            <div className="shippingAddress">
-              <p className="vads-u-margin-y--0">
-                {shippingAddress.street} {shippingAddress.street2 || ''}
+          </section>
+          <PrintDetails />
+        </>
+      )}
+      {selectedProductArray.length <= 0 && (
```
Comment (bradl3yC) [inline src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx:172]: We don't need to check for negative length

File: src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -74,94 +75,135 @@ const ConfirmationPage = ({
...
+                </h4>
+                <p className="vads-u-margin--0">
+                  for {fullName.first} {fullName.last}
+                </p>
+                <p className="vads-u-margin-bottom--0">
+                  <strong>Items ordered</strong>
+                </p>
+                <ul className="vads-u-margin-bottom--1">
+                  {selectedProductArray.map(product => (
+                    <li key={product.productId}>
```
Comment (bradl3yC) [inline src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx:112]: Do you think its worth while to add optional chaining to nullsafe all of these objects we're accessing?

**Review by bradl3yC — APPROVED** [review]

File: src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx (hunk 2 above)
Comment (mr0sari0) [reply src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx:112]: Yeah, I agree with extra protection for our objects!

**Comment by mr0sari0** [comment]
Unit tests that are not relating to this PR are failing. Started a thread in the #vfs-platform-support channel [here](https://dsva.slack.com/archives/CBU0KDSB1/p1593106215334100).

**Pull request merged**